tests/jobs.py

```diff
@@ -901,80 +901,6 @@ def do_schedule(function, function2):
         do_schedule()
         self.assertEquals(2, len(executor.queued_tasks))
 
-    def test_retry_still_in_executor(self):
```

Comment:
Gotcha, I found and fixed the bug (which was unrelated to this PR), looks like it was caused by a merge conflict between my massive PR (dependency engine) and your massive PR (refactoring dagruns).